File: TimMovie/TimMovie.Infrastructure/Database/Models/Subscribe.cs
using System.ComponentModel.DataAnnotations;

namespace TimMovie.Infrastructure.Database;

public class Subscribe
{
    public Guid Id { get; set; }
    
    [Required]
    [MaxLength(70)]
    public string Name { get; set; }
    
    [Required]
    public float Price { get; set; }

    public string? Description { get; set; }
    
    public List<Film> Films { get; set; }
}

File: TimMovie/TimMovie.Admin/server/entities/Subscribe.ts
import { Column, Entity, Index, ManyToMany, OneToMany } from "typeorm";
import { Film } from "./Film";
import { Genre } from "./Genre";
import { UserSubscribe } from "./UserSubscribe";

@Index("PK_Subscribes", ["id"], { unique: true })
@Entity("Subscribes", { schema: "public" })
export class Subscribe {
  @Column("uuid", { primary: true, name: "Id" })
  id: string;

  @Column("character varying", { name: "Name", length: 70 })
  name: string;

  @Column("real", { name: "Price", precision: 24 })
  price: number;

  @Column("text", { name: "Description", nullable: true })
  description: string | null;

  @Column("boolean", { name: "IsActive", nullable: false, default: false })
  isActive: boolean;

  @ManyToMany(() => Film, (films) => films.subscribes)
  films: Film[];

  @ManyToMany(() => Genre, (genres) => genres.subscribes)
  genres: Genre[];

  @OneToMany(() => UserSubscribe, (userSubscribes) => userSubscribes.subscribe)
  userSubscribes: UserSubscribe[];
}